src/core/purify.js

```diff
@@ -7,9 +7,9 @@ export default purify;
 
 purify.addHook('afterSanitizeAttributes', (node) => {
   // Set all elements owning target to target=_blank
-  // and add rel="noreferrer".
+  // and add rel="noopener noreferrer".
   if ('target' in node) {
     node.setAttribute('target', '_blank');
-    node.setAttribute('rel', 'noreferrer');
+    node.setAttribute('rel', 'noopener noreferrer');
```

Comment:
I did this to be consistent with the other usage lint enforces. Certainly for FF noreferrer does what noopener does but adding both might be more bullet proof if the former gets deprecated or some other browser deals with noreferrer differently.